server/openapi.yml

```diff
@@ -461,8 +459,6 @@ components:
   responses:
     NoContent:
       description: リクエストが成功し、返すリソースがない時のレスポンス
-      content:
-        {}
```

Comment:
> Empty Response Body
Some responses, such as 204 No Content, have no body. To indicate the response body is empty, do not specify a content for the response:

https://swagger.io/docs/specification/describing-responses/